data/learn.yaml

```diff
@@ -1,9 +1,25 @@
 - title: "Merging OpenTracing and OpenCensus: Goals and Non-Goals"
   url: https://medium.com/opentracing/merging-opentracing-and-opencensus-f0fe9c7ca6f0
-  date: 03-28-2019
+  source:
+    title: The OpenTracing blog
+    url: https://medium.com/opentracing
 - title: "Merging OpenTracing and OpenCensus: A Roadmap to Convergence"
   url: https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289
-  date: 04-19-2019
+  source:
+    title: The OpenTracing blog
```

Comment:
Do we want to call this OpenTracing blog?

Reply:
What else would you suggest calling it?

Reply:
@bogdandrutu yes, this article was posted on OT blog and was references from OC blog entry at opencensus.io/blog